openlibrary/macros/SearchResultsWork.html

```diff
@@ -106,9 +106,9 @@ <h3 itemprop="name" class="booktitle">
               $if len(doc.get('ia')) > 1:
                 $ blur_preview = "preview-covers--blur" if blur else ""
                 <span class="preview-covers $blur_preview">
-                  $for x, i in enumerate(doc.get('ia')[1:10]):
+                  $for x, i in enumerate(doc.get('ia')[1:5]):
                     <a href="$(book_url)?edition=ia:$(urlquote(i))">
-                      <img width="30" height="45" loading="lazy" src="//archive.org/services/img/$i" alt="$_('Cover of edition %(id)s', id=i)">
+                      <img class="$_('preview-covers-img-%(count)s', count=x)" width="30" height="45" loading="lazy" src="//archive.org/services/img/$i" alt="$_('Cover of edition %(id)s', id=i)">
```

Comment:
I don't think we want this class to be internationalized, just the alt tag.

The `_` function does i18n (internationalization) of the string

You probably just want:


 ```suggestion
                      <img class="preview-covers-img-$x" width="30" height="45" loading="lazy" src="//archive.org/services/img/$i" alt="$_('Cover of edition %(id)s', id=i)">
```